tests/TestCase.php

```diff
@@ -2,6 +2,34 @@
 
 namespace ipl\Tests\Web;
 
+use ipl\Html\ValidHtml;
+use PHPUnit\Util\Xml;
+
+// phpcs:disable
+if (class_exists('PHPUnit_Util_XML')) {
+    // Support older PHPUnit versions
+    class_alias('PHPUnit_Util_XML', 'PHPUnit\\Util\\Xml');
+}
+
 abstract class TestCase extends \PHPUnit\Framework\TestCase
 {
+    /** Assert that HTML is equal */
```

Comment:
I believe you can re-use the docstring from ipl-html, no?

Reply:
The [`asertHtml`](https://github.com/Icinga/ipl-html/blob/master/tests/TestCase.php#L18-L23) from ipl-html did not have types for everything. I added the missing Types, so the PHPDocs would become redundant because they are already present in the Method Signature. I can add them back If we want them regardless, the actual comment stayed the same.